.docs/user-guide/config.md

```diff
@@ -256,3 +256,22 @@ supported:
 Docker   | docker
 
 The volume driver `docker` is automatically activated.
+
+### VOLUME MOUNT PRE-EMPTION
+There is a capability to pre-emptively detach any existing attachments to other
+instances before attempting a mount.  This will enable use cases for
+availability where another instance must be able to take control of a volume
+without the current owner instance being involved.  The operation is considered
+equivalent to a power off of the existing instance for the device.
+
+Property Name | Environment Variable | CLI Flag
+--------------|----------------------|-------------
+`volume.mountpreempt`   | `REXRAY_VOLUME_MOUNTPREEMPT`          | `--preempt`
```

Comment:
Not sure I am following what you're saying completely.

How would you rather see it if it's a vdm defined flag?

```
---
rexray:
  storageDrivers:
  - openstack
openStack:
  authUrl: url
  username: clintonskitson
  password: pass
  tenantName: 10677883784544-Project
  regionName: region-a.geo-1
volume:
  mountPreEmpt: true
```

```
---
rexray:
  storageDrivers:
  - openstack
  volume:
    mountPreEmpt: true
openStack:
  authUrl: url
  username: clintonskitson
  password: pass
  tenantName: 10677883784544-Project
  regionName: region-a.geo-1
```